From: Jakob Schiotz <schiotz@fysik.dtu.dk>
Date: Wed Aug 13 15:44:38 CEST 2025

Fixes an error in the configure script, where a test program was
incorrect.  With the new GCC this triggers an error.

Also adapt to the MPI part of FFTW3 now being in a separate package.

--- configure.old	2025-08-12 19:37:21.720900286 +0200
+++ configure	2025-08-12 19:39:03.242292333 +0200
@@ -21917,7 +21917,7 @@
 
       int *argc;
       char **argv;
-      MPI_Init(argc, argv);
+      MPI_Init(argc, &argv);
 
   ;
   return 0;
@@ -22318,8 +22318,8 @@
     { printf "%s\n" "$as_me:${as_lineno-$LINENO}: using EasyBuild parameters to set FFTW3" >&5
 printf "%s\n" "$as_me: using EasyBuild parameters to set FFTW3" >&6;}
     vdw_fftw3_type="ebm"
-    vdw_fftw3_incs="-I${EBROOTFFTW}/include"
-    vdw_fftw3_libs="-L${EBROOTFFTW}/lib -lfftw3_mpi -lfftw3_threads -lfftw3f -lfftw3"
+    vdw_fftw3_incs="-I${EBROOTFFTW}/include -I${EBROOTFFTWMPI}/include"
+    vdw_fftw3_libs="-L${EBROOTFFTW}/lib -L${EBROOTFFTWMPI}/lib -lfftw3_mpi -lfftw3_threads -lfftw3f -lfftw3"
   fi
 fi
 
